tests/unit/drivers/test_concat_driver.py

```diff
@@ -9,16 +9,16 @@
 def input_fn():
     doc1 = Document()
     doc1.id = 1
-    doc1.embedding.CopyFrom(array2pb(np.random.random([7]).astype('float32')))
+    doc1.embedding.CopyFrom(array2pb(np.random.random([7])))
     c = doc1.chunks.add()
     c.id = 3
-    c.embedding.CopyFrom(array2pb(np.random.random([5]).astype('float32')))
+    c.embedding.CopyFrom(array2pb(np.random.random([5])))
     doc2 = Document()
     doc2.id = 2
-    doc2.embedding.CopyFrom(array2pb(np.random.random([3]).astype('float32')))
+    doc2.embedding.CopyFrom(array2pb(np.random.random([3])))
     d = doc2.chunks.add()
     d.id = 4
-    d.embedding.CopyFrom(array2pb(np.random.random([9]).astype('float32')))
+    d.embedding.CopyFrom(array2pb(np.random.random([9])))
```

Comment:
a perfect place to use parameterized test :)

Reply:
can u develop this idea in a small example PR maybe? No rush

Reply:
sure